src/__test__/components/PlatformError.test.jsx

```diff
@@ -0,0 +1,178 @@
+import React from 'react';
```

Comment:
This new test file is unrelated to the functionality in this PR, but it was necessary to avoid having problems with codecov